api/src/test/java/org/pmiops/workbench/rdr/RdrExportServiceImplTest.java

```diff
@@ -205,6 +205,21 @@ public void exportWorkspace() throws ApiException {
     verify(mockRdrApi).exportWorkspaces(Arrays.asList(rdrWorkspace), NO_BACKFILL);
   }
 
+  @Test
+  public void exportWorkspace_backfill() throws ApiException {
+    List<Long> workspaceID = new ArrayList<>();
+    workspaceID.add(1l);
+    RdrWorkspace rdrWorkspace = toDefaultRdrWorkspace(mockWorkspace);
+    when(rdrMapper.toRdrModel(mockWorkspace)).thenReturn(rdrWorkspace);
+    rdrExportService.exportWorkspaces(workspaceID, true);
+    verify(mockWorkspaceService)
+        .getFirecloudUserRoles(
+            mockWorkspace.getWorkspaceNamespace(), mockWorkspace.getFirecloudName());
+    verify(rdrExportDao, times(1)).saveAll(anyList());
```

Comment:
if we are backfilling this should not have been called

Reply:
Yep, updated along with code